src/utils/promisify.js

```diff
@@ -2,6 +2,6 @@ import Promise from 'pinkie';
 import pify from 'pify';
 
 
-export default function (fn) {
-    return pify(fn, Promise);
+export default function (fn, multiArgs) {
```

Comment:
I think it's better to use the native `Promise` and `util.promisify`.

`child_process.exec` has a special native promisified version, which resolves `{ stdout, stderr }` as a child process results. It's perfect for our use case/

Reply:
As we decide to always use PowerShell, we don't need this anymore.